array-from.js

```diff
@@ -67,6 +67,7 @@ if (!Array.from) {
 				}
 				defineProperty(A, k, {
 					'value': mappedValue,
+					'writeable': true,
```

Comment:
It looks like `writeable` is still present in the diff. Could you please remove it, and rebase this branch down to a single commit?

Otherwise, it looks great, thanks!